hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -161,6 +161,10 @@ public void handle(@NonNull final HandleContext handleContext) {
         builder.sequenceNumber(topic.sequenceNumber());
         builder.runningHash(topic.runningHash());
         builder.deleted(topic.deleted());
+        // TODO: added so unit tests pass (fix this when implementing the actual logic)
+        builder.feeScheduleKey(topic.feeScheduleKey());
+        builder.freeMessagesKeyList(topic.freeMessagesKeyList());
+        builder.customFees(topic.customFees());
```

Comment:
On update, we should not set the field if the transaction does not have new data.
(e.g. if schedule key not set, do not set it here; if custom fees is empty, do not replace the field here).

Side note: I do not know if we've defined how to remove all fee schedules, given that PBJ protobuf repeated fields do not have an "unset" condition different from "empty list".